Library/Homebrew/rubocops/uses_from_macos.rb

```diff
@@ -82,6 +82,7 @@ class UsesFromMacos < FormulaCop
           gzip
           openssl
           openssl@1.1
+          openssl@3
```

Comment:
This addition is incorrect. `openssl@3` is not provided by macOS.

Reply:
Neither is OpenSSL 1.1.

Reply:
@Bo98 Yup. Technically that shouldn't be here either and we should fix the underlying issues which make `uses_from_macos "openssl"` work as expected. Probably warrants a `TODO` comment until that point, too.